src/xunit.performance.api/XunitPerformanceHarnessOptions.cs

```diff
@@ -15,6 +15,7 @@ internal sealed class XunitPerformanceHarnessOptions
         public XunitPerformanceHarnessOptions()
         {
             _outputDirectory = Directory.GetCurrentDirectory();
+            _tmpDirectory = Directory.GetCurrentDirectory();
```

Comment:
You should probably make a random directory in the output directory.  This ensures that you can easily delete it and that we don't inadvertently mess things up in a directory that we didn't create.